app/services/facebook_service.rb

```diff
@@ -14,4 +14,30 @@ def client
   def uids
     client.get_connections(:me, :friends).map { |friend| friend['id'] }
   end
+
+  def share!(checkin)
+    if checkin.facebook_url_hash.blank?
+      checkin.update_column(:facebook_url_hash, checkin.generate_url_hash)
+    end
+
+    work_title = checkin.episode.work.title
+    episode_number_title = checkin.episode.number_title
+    title = "#{work_title} #{episode_number_title}".truncate(30)
+
+    message = checkin.comment.squish.truncate(50) if checkin.comment.present?
+    link = if Rails.env.development?
+             "http://www.annict.com/checkins/redirect/fb/#{checkin.facebook_url_hash}"
+           else
+             "#{ENV['ANNICT_URL']}/checkins/redirect/fb/#{checkin.facebook_url_hash}"
+           end
+    name = I18n.t("checkins.facebook_share_text", title: title)
+    caption = "Checkined by #{checkin.user.username}"
+
+    client.put_connections("me", "feed",
+      message:      message,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.